benchmarks/zap_test.go

```diff
@@ -34,20 +34,21 @@ import (
 var (
 	errExample = errors.New("fail")
 
-	_messages   = fakeMessages(1000)
-	_tenInts    = []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 0}
+	_messages = fakeMessages(1000)
+	// Go optimizes away allocations for numbers below 256
+	_tenInts    = []int{1001, 1002, 1003, 1004, 1005, 1006, 1007, 1008, 1009, 1010}
```

Comment:
Fair point. If you're still experimenting with this PR, you can probably get this specific change merged as a separate PR.

Reply:
thanks btw, how / on which machine do you run `make updatereadme` as the absolute (and seemingly even some times the relative numbers) depends on the hw/arch/os it's being ran on ?

Reply:
I think original runs were on Intel Macbook Pros, and since then on Apple Silicon Macbook Pros.
Absolute numbers are less important than the relative performance, so as long as the numbers are consistent and the benchmarks were all run on the same machine, it should be good to check-in.